src/test/rgw/bucket_notification/test_bn.py

```diff
@@ -2266,111 +2204,38 @@ def test_ps_s3_multipart_on_master_http():
 
     # check http receiver
     keys = list(bucket.list())
-    print('total number of objects: ' + str(len(keys)))
-    events = http_server.get_and_reset_events()
-    for event in events:
-        assert_equal(event['Records'][0]['opaqueData'], opaque_data)
```

Comment:
* we probably need a separate test for the `opaque_data`
* i think we are allready testing etags elsewhere in a better way

Reply:
got it